dataactcore/scripts/raw_sql/backfill_funding_office_names_q12019.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE published_award_financial_assistance AS pafa
+SET funding_office_name = office.office_name
+FROM office
+WHERE pafa.funding_office_code = office.office_code
+  AND cast_as_date(pafa.action_date) >= '2018/10/01' AND cast_as_date(pafa.action_date) < '2018/01/01'
```

Comment:
I think one of these dates is wrong. Shouldn't this be `'2019/01/01'`? Also, do we want to make this date a little later since this feature won't be going out exactly on the 1st?